Templates/Files/MS SQL/MS SQL database SELECT to CSV file/long-description.md

```diff
@@ -0,0 +1,23 @@
+This template generates CSV data from Microsoft SQL Server. 
+Using this template, you can create a CSV file by fetching data from Microsoft SQL Server, converting it into JSON, and then transforming it into CSV file which will be saved on SFTP server.
```

Comment:
You can delete the part about JSON, as it is not important.
Also you you 'create' there, so lets use the same word form for 'transform' (instead of transforming), so something like "create CSV... ... , transform it into CSV format and write it to an SFTP server."